component/main.jsonnet

```diff
@@ -1,10 +1,110 @@
 // main template for csi-driver-smb
+local com = import 'lib/commodore.libjsonnet';
+local espejo = import 'lib/espejo.libsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.csi_driver_smb;
 
-// Define outputs below
+local pvs = [
+  kube.PersistentVolume(pv.name) {
+    metadata+: {
+      namespace: pv.namespace,
+    },
```

Comment:
PersistentVolumes are cluster-scoped, it's not necessary to set `metadata.namespace`, and if set, the field doesn't have an effect.

Reply:
Removed it